src/components/externalLinks/ExternalLinksCRUD/ConfigureLinkAction.tsx

```diff
@@ -0,0 +1,185 @@
+                        styles={{
+                            ...multiSelectStyles,
+                            ...customMultiSelectStyles,
+                            option: (base, state) => ({
+                                ...customMultiSelectStyles.option(base, state),
+                                backgroundColor: state.isSelected
+                                    ? 'var(--B100)'
+                                    : state.isFocused
+                                    ? 'var(--N100)'
+                                    : 'white',
+                                color: state.isSelected ? 'var(--B500)' : 'var(--N900)',
+                            }),
+                            container: (base, state) => ({
+                                ...base,
+                                marginTop: '6px',
+                            }),
+                            control: (base, state) => ({
+                                ...customMultiSelectStyles.control(base, state),
+                                width: '150px',
+                            }),
+                            valueContainer: (base) => ({
+                                ...base,
+                                padding: '0 10px',
+                            }),
+                            placeholder: (base) => ({
+                                ...base,
+                                color: 'var(--N500)',
+                            }),
+                        }}
+                    />
+                    {link.invalidTool && getErrorLabel('tool')}
+                </div>
+                <div className="link-name mr-12">
+                    <label>Name*</label>
+                    <input
+                        placeholder="Enter name"
+                        value={link.name}
+                        onChange={(e) => onNameChange(index, e.target.value)}
+                    />
+                    {link.invalidName && getErrorLabel('name')}
+                </div>
+                <div className="link-clusters mr-12">
+                    <span>Clusters*</span>
+                    <ReactSelect
+                        placeholder="Select clusters"
+                        name={`link-clusters-${index}`}
+                        value={selectedClusters}
+                        options={clusters}
+                        formatOptionLabel={formatOptionLabelClusters}
+                        onChange={(selected) => onClusterSelection(index, selected)}
+                        isMulti={true}
+                        hideSelectedOptions={false}
+                        closeMenuOnSelect={false}
+                        inputValue={clusterSearchInput}
+                        onBlur={() => {
+                            setClusterSearchInput('')
+                        }}
+                        onInputChange={(value: string, actionMeta: InputActionMeta) => {
+                            if (actionMeta.action === 'input-change') {
```

Comment:
we should extract it out to avoid inline function

Reply:
will be covered in the next PR.